content/en/tracing/trace_collection/library_config/dotnet-core.md

```diff
@@ -73,6 +73,23 @@ Tracer.Configure(settings);
 
 {{% /tab %}}
 
+{{% tab "appsettings.json" %}}
+
+To configure the tracer using an `appsettings.json` file, use the `<appSettings>` section. For example:
```

Comment:
After some further discussion, I realized that we don't read `appsettings.json` for Datadog configuration, so we should remove this. What we should focus on is removing mentions of `app.config`/`web.config` as we don't read those for Datadog configuration